### PR TITLE
fix(linux): Fix autopkgtests 🍒

### PR DESCRIPTION
**Pull request by ermshiperete**
Autopkgtests couldn't find pkg-config, so this change adds that as test dependency.

(🍒 picked from PR #8180 )

@keymanapp-test-bot skip

### CODE REVIEW

File: linux/debian/control [hunk 1]
```diff
@@ -37,8 +37,8 @@ Build-Depends:
  metacity,
  gawk,
 Standards-Version: 4.6.1
-Vcs-Git: https://github.com/keymanapp/keyman.git [linux/debian]
-Vcs-Browser: https://github.com/keymanapp/keyman/tree/master/linux/debian
+Vcs-Git: https://github.com/keymanapp/keyman.git -b stable-16.0 [linux/debian]
+Vcs-Browser: https://github.com/keymanapp/keyman/tree/stable-16.0/linux/debian
```
Comment (darcywong00) [inline linux/debian/control:41]: Should these stay master?

Comment (ermshiperete) [reply linux/debian/control:41]: I think it doesn't matter. We don't use this information anywhere, it's just used on Debian with their packaging infrastructure. Keeping the file content the same on stable and master simplifies cherry-picking IMO.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.45-alpha](https://keyman.com/downloads/releases/alpha/17.0.45)

**Pull request merged**